packaging/aleph-vm/DEBIAN/control

```diff
@@ -1,5 +1,5 @@
 Package: aleph-vm
-Version: 0.1.8
+Version: 0.2.5-26-g358c9ef
```

Comment:
Why commit this ?

Reply:
My bad, I'll revert it. It's a bit annoying that it can even be committed at all.